### DISCUSSION TITLE
Не работают субдомены

### DISCUSSION THREAD

**Discussion by Joy096** [AdguardTeam/AdGuardHome#4743] [General]
Здравствуйте. Столкнулся с такой проблемой, что не работают субдомены DoT для аутентификации устройств.
AdGuard Home установлен на Ubuntu 20.04, ARMx64 Oracle VPS , DNS использую в связке с CloudFlare.

Сертификат и ключ делал по этой инструкции
https://blog.thatopsguy.com/2021/08/lets-encrypt-cert-for-adguard-home/

В настройках Android DoT в формате example.com работает нормально, но при попытке установки субдомена (например: xiaomi.example.com) выдает ошибку. Порты в ufw открыты.

**Comment by ameshkov** [comment]
Скорее всего сертификат криво создали.

А вот тут лучше прочитайте, там про все это есть:
https://adguard.com/ru/blog/adguard-home-on-public-server.html

**Reply by Joy096** [reply]
Хоть дело и не в сертификате но все равно помогли. Нужно было запись "A" создать со звездочкой, это я не учёл!)
